Exercises/3-by-reference.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
-const inc = null;
+const inc = num => {
+  if (typeof num === 'object') ++num.n;
```

Comment:
If you want to implement strict checking, why don't check field `n` existence and typeof n?

Reply:
I agree. Verification was expected:
```const inc = num => {
  if (typeof num === 'object' && typeof num.n === 'number') ++num.n;
};
```
Getting an ```Error: Solution is too long```,  I decided to omit the type check.

If verification in this case is optional, can I not give it?